### ISSUE TITLE
Allow T01 and T03 CTAs to use the same wrapper markup

### ISSUE THREAD

**Issue by imcbride** [middlebury/frontend#27]
Currently, the CTA row on the T01 template uses `partials/card.twig` and the T03 template uses `partials/callout.twig`. In Drupal, these will be rendered on the page using either this for T01:

```
{% if content.field_calls_to_action|render|trim is not empty %}
  <div class="callout-group">
    <div class="callout-group__row">
      {% for cta in content.field_calls_to_action['#items'] %}
        {{ drupal_block(cta.plugin_id) }}
      {% endfor %}
    </div>
  </div>
{% endif %}
```

Or this for T03:

```
<div class="callout-group">
  <div class="callout-group__row">
    {% if content %}
      {{ content }}
    {% endif %}
  </div>
</div>
```

In each case, the interior markup of each block should be independent of the wrapper so we have flexibility over the content and presentation. Ideally, the wrapper markup would be similar between the two, though this isn't a strict requirement as long as the interior doesn't rely on it.

**Comment by zebapy** [comment]
@imcbride so this is still a problem even with the CTAs on the homepage and bottom of other pages using the same markup, right? https://github.com/middlebury/frontend/blob/develop/src/templates/partials/callout-group.twig

**Comment by imcbride** [comment]
If possible, I'd like to use the same wrapper divs for each. Currently, the homepage uses this:

```
    <div class="row text-center text-md-left section">
      <div class="col-md-6 col-lg-3 border-md-r mb-4 mb-lg-0">
        <div class="border-b border-md-b-0 pb-4 pb-md-0">
```

And the other pages use this:

```
<div class="callout-group">
  <div class="callout-group__row">
    <div class="callout-group__item">
```

**Comment by zebapy** [comment]
Thanks. Just wanted to clarify. Ari is working on homepage and I'll be revisiting this soon.

**Comment by zebapy** [comment]
closed in changes for #75
